src/ng/sniffer.js

```diff
@@ -61,7 +65,7 @@ function $SnifferProvider() {
       // so let's not use the history API also
       // We are purposefully using `!(android < 4)` to cover the case when `android` is undefined
       // jshint -W018
-      history: !!($window.history && $window.history.pushState && !(android < 4) && !boxee),
+      history: !!(hasHistoryPushState && !(android < 4) && !boxee),
```

Comment:
Plus, if I have `isChromePackagedApp` and `hasHistoryPushState` close together, it's easier for reference wrt to the comment explaining why Chrome Packaged Apps need special handling wrt to checking the existence of `history.pushState`.